code/sum.py

```diff
@@ -95,17 +94,22 @@ def get_summary(filename="1.mp4", subtitles="1.srt"):
     summary.to_videofile( output, codec="libx264", temp_audiofile="temp.m4a", remove_temp=True, audio_codec="aac")
     return True
 
-def download_video(url):
-    ''' Downloads specified Youtube video
+# download video with subtitles
+def downvdo_subs(subs):
+    ''' Downloads specified Youtube video's subtitles as a vtt/srt file.
     args:
-        url(str): Full url for youtube video
+        subs(str): Full url of Youtube video
     returns:
         True
     '''
-    yt = YouTube(url)
-    yt.set_filename('1')
-    video = yt.filter('mp4')[-1]
-    video.download(os.getcwd())
+    ydl_opts = {
+            'outtmpl': '1.%(ext)s',
+            'subtitlesformat':'srt',
+            'writeautomaticsub':True
+            }
+
+    with youtube_dl.YoutubeDL(ydl_opts) as ydl:
+        ydl.download([subs])
```

Comment:
Add comments for the following:
What will be the filename and location of the downloaded video and subtitle? Will it ensure the highest resolution as resolved in a previous pull request?

These details are required for further development.

Reply:
@AdiChat added the comments, as requested.